### PR TITLE
fix(index.md): typos and wording

### PR DESCRIPTION
**Pull request by cbontems**
### Description

Fixes a few typos and words for better clarity.

### Motivation

I had to turn to the original version in english to make sure I understood. So I guess this may help others also.

### CODE REVIEW

File: files/fr/web/css/@property/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ slug: Web/CSS/@property
 
 {{CSSRef}}{{SeeCompatTable}}
 
-La règle {{cssxref("at-rule")}} [CSS](/fr/docs/Web/CSS) **`@property`** fait partie du panel d'API [CSS Houdini](/fr/docs/Web/Houdini), qui permet de définir explicitement leurs propriétés CSS personnalisées (voir {{cssxref('--*', 'CSS custom properties')}}). Cela permet la vérification des types de propriétés, de leur donner des valeurs par défaut et de déterminer si une propriété peut hériter ses valeurs ou non.
+La règle {{cssxref("at-rule")}} [CSS](/fr/docs/Web/CSS) **`@property`** fait partie du panel d'API [CSS Houdini](/fr/docs/Web/Houdini), qui permet aux développeurs de définir explicitement leurs propriétés CSS personnalisées (voir {{cssxref('--*', 'CSS custom properties')}}). Cela permet la vérification des types de propriétés, de leur donner des valeurs par défaut et de déterminer si une propriété peut hériter ses valeurs ou non.
```
Comment (cw118) [inline files/fr/web/css/@property/index.md:8]: Related to/similar to the [mozfr guide](https://github.com/mozfr/besogne/wiki/Guide-stylistique-pour-la-traduction#sadresser-%C3%A0-lutilisateur-ou-lutilisatrice) :)

```suggestion
La règle {{cssxref("at-rule")}} [CSS](/fr/docs/Web/CSS) **`@property`** fait partie du panel d'API [CSS Houdini](/fr/docs/Web/Houdini), qui permet aux développeuses et développeurs de définir explicitement leurs propriétés CSS personnalisées (voir {{cssxref('--*', 'CSS custom properties')}}). Cela permet la vérification des types de propriétés, de leur donner des valeurs par défaut et de déterminer si une propriété peut hériter ses valeurs ou non.
```

**Review by cw118 — CHANGES_REQUESTED** [review]
Thank you for the PR @cbontems ! Only one small suggestion :)

File: files/fr/web/css/@property/index.md (hunk 1 above)
Comment (cbontems) [reply files/fr/web/css/@property/index.md:8]: Great suggestion!

**Review by cw118 — APPROVED** [review]
Thanks very much for these corrections @cbontems !

**Pull request merged**